src/components/wordlistAPI.svelte

```diff
@@ -14,7 +16,7 @@
     }
 </script>
 
-<form on:submit|preventDefault={onSubmit}>
+<form on:submit|preventDefault={text && onSubmit}>
```

Comment:
from a UX perspective, it might be better to do this treatment on the button instead, e.g., `disabled={! text}`

Reply:
I like this idea. It looks like I just add disabled={! text} to the button element.

Reply:
Added it to the input element.